src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -17,7 +17,8 @@ gregtech.top.transform_up=§cStep Up§r %s (%dA) -> %s (%dA)
 gregtech.top.transform_down=§aStep Down§r %s (%dA) -> %s (%dA)
 gregtech.top.transform_input=§6Input:§r %s (%dA)
 gregtech.top.transform_output=§9Output:§r %s (%dA)
-gregtech.top.fuel_info=/%,d %,d s
+gregtech.top.fuel_info=/%,d for %,d:%02d mins
```

Comment:
I think it's pretty understandable that it's in measured in minutes
```suggestion
gregtech.top.fuel_info=/%,d for %,d:%02d
```

Reply:
12:45 Is it mins and secs? It could be hours and mins. A newbie isn't going to know.

Reply:
How about `0:13 mins`? I think it's 13 minutes.

Reply:
A newbie will see that it updates each second and will easily understand the measurement of time

Reply:
Basically `mins` doesn't solve the problem you are talking about because it's still may be **mins** and secs or hours and **mins**.

Reply:
can you put the comments in the pr rather than a converstion where it is easier for others to see and discuss.

personally I find cosmetics one of the least interesting parts of software development and don't want to waste any time discussing it